{
  "FaultLabel": "Resolved power cut",
  "Type": "ResolvedFault",
  "Icon": "icon-success",
  "ShowViewOnMapCTA": false,
  "UnderReview": false,
  "AdditionalFaultInfo": "Good news! The electricity in your area has been restored. The power cut was caused by an unexpected incident on our underground cable that provides your supply.",
  "CTAs": [],
  "faultNumber": "60494209",
  "multipleFaultNumbers": null,
  "date": "2020-10-16T18:05:09",
  "region": "Carnforth",
  "faultType": "Resolved Fault",
  "faultStatus": "Resolved",
  "previousFaultStatus": "Resolved",
  "consumersOff": 0,
  "estimatedTimeOfRestoration": null,
  "estimatedTimeOfRestorationMajority": "2020-10-16T18:05:09",
  "information": "",
  "actualTimeOfRestoration": "2020-10-16T18:05:09",
  "outageCentrePoint": {
    "lat": 54.112041,
    "lng": -2.786689
  },
  "outageLocations": null,
  "AffectedPostcodes": " LA5 8DA, LA5 8DB, LA5 8DD",
  "EtrMUnderReview": true,
  "WebTMSFaultType": "U/GM"
}